crates/forge/src/collecting.rs

```diff
@@ -27,8 +28,21 @@ impl TestsFromCrate {
             }
         }
         Self {
-            sierra_program: self.sierra_program.clone(),
-            test_cases: result.into_iter().cloned().collect(),
+            sierra_program: self.sierra_program,
+            test_cases: result,
+            test_crate_type: self.test_crate_type,
+        }
+    }
+
+    pub fn extract_ignored(self) -> Self {
```

Comment:
Does it need to be `pub` if we only use it in this file? I'd say it's mostly an implementation detail - does not need to be a part of the public api.